### PR TITLE
[4.0] Workflow string

### PR DESCRIPTION
**Pull request by brianteeman**
Updating #23992 for workflow v3

Pull Request for Issue #29547

### CODE REVIEW

**Comment by richard67** [comment]
Harald has a holiday. Not sure if he can review soon. Maybe @bembelimen can?

Update: Maybe it was an automatic review request because of code ownership. Sorry for disturbing.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 30b842b5dc172f8cedc8cb894f45d929f5c4d9c3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29554">issues.joomla.org/tracker/joomla-cms/29554</a>.</sub>

File: administrator/language/en-GB/com_workflow.sys.ini [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 COM_WORKFLOW="Workflows"
 COM_WORKFLOW_BASIC_WORKFLOW="Basic Workflow"
+COM_WORKFLOW_BASIC_STAGE="Basic Stage"
```
Comment (Quy) [inline administrator/language/en-GB/com_workflow.sys.ini:8]: Please reorder.

**Comment by richard67** [comment]
Maybe I understand something wrong: I have applied the change of this PR, but I still see the untranslated strings when viewing a group's permission like described in issue #29547 .

Anyone can enlighten me?

**Comment by Quy** [comment]
PR is working for me.

**Comment by richard67** [comment]
@Quy Just by applying the new language string?

**Comment by Quy** [comment]
Yes just applying the PR.

**Comment by Quy** [comment]
Before PR:
29554-before

**Comment by richard67** [comment]
Doesn't work here, I get mad. I mean the PR is right, no doubt, just I apply it and still see those language constants in the groups' permissions.

**Comment by richard67** [comment]
For me both the basic stage and the basic workflow are not translated, like it is shown in the screenshot in issue #29547 . Wait, let's update my 4.0-dev branch, it is missing last 10 commits.

**Comment by Quy** [comment]
Sorry I don't know why it is not working for you.

**Comment by richard67** [comment]
@Quy In **users** permissions all is fine here, only in **groups** permissions not. Sure you've checked **groups** and not users? Your screenshot doesn't show it.

**Comment by Quy** [comment]
@richard67 Oops, you are correct.

**Comment by richard67** [comment]
@Quy So we have dilemma: This PR here is correct because it fixes the one untranslated language constant in the **users'** permisson, but it doesn't solve the issue it refers to because that was talking about the two untranslated constants in the **groups** permission.

What to do now? Test this PR with success and reopen the issue?

**Comment by Quy** [comment]
Let's wait for Brian, Herald or JM to chime in.

**Comment by richard67** [comment]
I think it's some PHP stuff that the groups permissons don't load that (sys) language file.

**Comment by brianteeman** [comment]
super easy change ;)

**Comment by richard67** [comment]
Yes, no brainer ;-)

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 46b3b23761d8a8c4f94fd3998b5892e0fbb04982<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29554">issues.joomla.org/tracker/joomla-cms/29554</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 46b3b23761d8a8c4f94fd3998b5892e0fbb04982<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29554">issues.joomla.org/tracker/joomla-cms/29554</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29554">issues.joomla.org/tracker/joomla-cms/29554</a>.</sub>

**Review by bembelimen — APPROVED** [review]

**Comment by infograf768** [comment]
tks

**Comment by brianteeman** [comment]
Thx

**Pull request merged**